app/Http/Controllers/UserController.php

```diff
@@ -3,17 +3,56 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
 
 class UserController extends Controller
 {
+    public function login()
+    {
+        return view('login.login');
+    }
+
+    public function postLogin(Request $request)
+    {
+        $datas = $request->only('email', 'password');
+ 
+        if (Auth::attempt($datas)) {
+            // if(Auth::user()->isAdmin){
+            //     return redirect()->route('home')->with('thongbao', 'Đăng nhập thành công');
+            // }
+            return redirect()->route('home')->with('thongbao', 'Đăng nhập thành công');
+        }
+
+        return redirect()->back()->withInput()->with('thongbao', 'Đăng nhập thất bại');
```

Comment:
```suggestion
            return redirect()->route('home')->with('thongbao', 'Đăng nhập thành công');
        }

        return redirect()->back()->withInput()->with('thongbao', 'Đăng nhập thất bại');
```
để đồng bộ ta dùng biến tiếng anh nha